config/environment.js

```diff
@@ -21,6 +21,7 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    validationEndpoint: process.env.VALIDATION_ENDPOINT,
```

Comment:
This doesn't use the Semantic Works stack yet, which is why I added it this way. It depends on a `.env` file in the root of the project with the correct endpoint url.

Reply:
So, do you want to update it before merging?

Reply:
I need to make changes to the dispatcher first (which I'm _trying_ to do now :smile:). But I don't see it as a blocker for merging. I can swap out the config later.

Reply:
Updated the dispatcher, so now it depends on this pr: https://github.com/lblod/app-poc-shacl-validation/pull/5